### PR TITLE
Create ui account

### LINKED ISSUE #5: Create UI: Create Account
Use Case: Create Account

1. Create UI to create an account
2. Make Post Call to App
3. Verify if Account is created succesfully

### PR DESCRIPTION
**Pull request by conorH22**
Created Login and Signup forms  for Account UI. with view controllers for the relevant pages

Closes #5

### CODE REVIEW

File: src/main/java/euteam/cleanmachine/service/UserService.java [hunk 1]
```diff
@@ -4,19 +4,41 @@
...
+    Logger log = LoggerFactory.getLogger(this.getClass());
 
     public List<User> findAll() {
 
         List<User> users = (List<User>) repository.findAll();
 
         return users;
     }
+    @RequestMapping(value="/users", method= RequestMethod.POST)
+    public String customerSubmit(@ModelAttribute User user, Model model) {
```
Comment (Raul6469) [inline src/main/java/euteam/cleanmachine/service/UserService.java]: Request mappings should not be declared in Service classes, but in a Controller instead

**Comment by conorH22** [comment]
Is there additional  labels to the signup form like username and  role in the userSignupdto.java file.
I'm getting  this error message when I submit signup form  

" There was an unexpected error (type=Bad Request, status=400).
Validation failed for object='userSignUpDto'. Error count: 3"

**Comment by conorH22** [comment]
After submitting the form. 

This error message is shown  

> "Therewas an unexpected error (type=UnsupportedMedia Type, status=415).
Content type 'application/x-www-form-urlencoded;charset=UTF-8' not supported "

I changed the signup form to now have the option to set username and  select role
[Link to changes](https://github.com/eu-team/clean-machine/pull/14/commits/bb0bc32ec1c4c647429028cbd336b0e8b06dc4c3#diff-c98bf22d9617befa7ff2799623721367L72)

**Comment by Raul6469** [comment]
Oh ok I see, you have to provide the data as a JSON, not as `x-www-form-urlencoded`. The payload has to follow this format:
```json
{
	"name": "Name",
	"username": "username",
	"email": "email@domain.com",
	"password": "password",
	"roleName": "ROLE_CUSTOMER"
}
```
If you can't make the request with a classic `<form>` tag, you will have to do it with JS by making an AJAX request.

**Comment by conorH22** [comment]
User Signup form works through Ajax post request and data goes to the database

**Pull request merged**